crates/types/src/nodes_config.rs

```diff
@@ -46,6 +46,7 @@ pub enum Role {
     Admin,
     /// Serves the metadata store
     MetadataStore,
+    #[cfg(feature = "replicated-loglet")]
     /// Serves a log server for replicated loglets
     LogServer,
```

Comment:
A bit off-topic: If we want to support rolling back from this version, then we need to teach `release-1.0` to ignore this role when deserializing it from the metadata-store. Right now it is panicking.

Reply:
Yes, there will definitely be an intermediate release before we can enable this. That release will be the safe fallback.